env.sh

```diff
@@ -99,7 +99,8 @@ EOF
   run_cmd cd "$cmssw_base/src"
   run_cmd eval `scramv1 runtime -sh`
   if ! [ -d "CombineHarvester" ]; then
-    run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    #run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    run_cmd bash <(https://raw.githubusercontent.com/nucleosynthesis/work-tools/master/sparse-checkout-SL-ssh.sh)
```

Comment:
- please use CombineHarvester from the central repository https://github.com/cms-analysis/CombineHarvester
- It looks like for the newest release, a sparse checkout is not needed, and one can directly use 
  ```
  git clone https://github.com/cms-analysis/CombineHarvester.git CombineHarvester
  ```